.gitignore

```diff
@@ -2,6 +2,7 @@
 # npm
 node_modules
 package-lock.json
+yarn.lock
```

Comment:
I'd prefer to commit the lockfiles for reproducibility. But since the `package-lock.json` is already ignored, @lynchjames must have weighed the pros and cons already.